src/test/java/uk/gov/hmcts/reform/unspec/model/ServedDocumentFilesTest.java

```diff
@@ -44,7 +44,8 @@ void shouldReturnMoreThanOneError_WhenBothParticularsOfClaimFieldsAreNotNull() {
                 .build();
 
             assertThat(servedDocumentFiles.getErrors())
-                .containsOnly("More than one Particulars of claim details added");
+                .containsOnly("You need to either upload 1 Particulars of claim only or enter the Particulars "
```

Comment:
if you update the code according to comments I made above, then you'll need to update the test (it's currently using `particularsOfClaimDocument` field, and should use the new one).

Reply:
Also, sonarqube might fail your build because of uncovered backwards compatible code. Let's not worry about it now, maybe it will pass. If not we could ignore specific files from coverage scan (it's pointless to cover backwards compatible code that's soon to be removed).